src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -69,6 +70,24 @@ class ParticipantProjectSpeciesController(
     return makeGetResponse(model)
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @PutMapping("/{participantProjectSpeciesId}")
+  @Operation(summary = "Updates a participant project species entry.")
+  fun updateParticipantProjectSpecies(
+      @PathVariable participantProjectSpeciesId: ParticipantProjectSpeciesId,
+      @RequestBody payload: UpdateParticipantProjectSpeciesPayload
+  ): SimpleSuccessResponsePayload {
+    participantProjectSpeciesStore.update(participantProjectSpeciesId) {
+      it.copy(
+          feedback = payload.participantProjectSpecies.feedback,
+          rationale = payload.participantProjectSpecies.rationale,
+          submissionStatus = payload.participantProjectSpecies.submissionStatus)
+    }
```

Comment:
The pattern I've been using is to put this `copy` in a function on the payload class, just so the assignments are closer to the field definitions and it's easier to spot if there's something missing/mismatched.